docs/pages/pmd/userdocs/tools/maven.md

```diff
@@ -0,0 +1,234 @@
+
+Given that the newer PMD version is compatible, you can override the PMD version, that the
+Maven plugin will use and benefit from the latest bugfixes and enhancements:
+
+``` xml
+<project>
+    <properties>
+        <pmdVersion>...choose your version...</version>
+    </properties>
+...
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-pmd-plugin</artifactId>
+                    <version>3.8</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>net.sourceforge.pmd</groupId>
+                            <artifactId>pmd-core</artifactId>
+                            <version>${pmdVersion}</version>
+                        </dependency>
+                        <dependency>
+                            <groupId>net.sourceforge.pmd</groupId>
+                            <artifactId>pmd-java</artifactId>
+                            <version>${pmdVersion}</version>
+                        </dependency>
+                        <dependency>
+                            <groupId>net.sourceforge.pmd</groupId>
+                            <artifactId>pmd-javascript</artifactId>
+                            <version>${pmdVersion}</version>
+                        </dependency>
+                        <dependency>
+                            <groupId>net.sourceforge.pmd</groupId>
+                            <artifactId>pmd-jsp</artifactId>
+                            <version>${pmdVersion}</version>
+                        </dependency>
+                    </dependencies>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+...
+</project>
+```
+
+### Reference
+
+For more information, please see the well documented PMD plugin project page here:
+<http://maven.apache.org/plugins/maven-pmd-plugin/index.html>.
+Also, the bug tracker for this plugin is [here](http://jira.codehaus.org/browse/MPMD).
+
+
+## Maven 1
+
+{% include warning.html content="Apache Maven 1.x has reached its end of life, and is no longer supported. For more information, see the [announcement](http://maven.apache.org/maven-1.x-eol.html). Users are encouraged to migrate to the current version of Apache Maven." %}
+
+This section is about the maven 1 PMD plugin.
+
+### Running the pmd plugin
+
+#### report
+
+To include the Maven report in the project reports section add the following line under
+the reports element in your project.xml:
+
+    <report>maven-pmd-plugin</report>
+
+This will add an entry to the 'project reports' section with the PMD report.
+
+##### manual
+
+To run PMD on a Maven project without adding it as a report, simply run
+
+    maven pmd xdoc
+
+The PMD plugin writes the report in XML which will then be formatted into more readable HTML.
+
+#### Customization
```

Comment:
If the structure for the "Maven 1" section is the same as the one for Maven 2 & 3 then there's one extra pound here too